analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -255,6 +258,16 @@ private VerifierBuilder CreateBuilder(ProjectType projectType, params string[] f
                 .WithProtobufPath(@$"{testRoot}\symrefs.pb");
         }
 
+        private static void VerifyReferences(IReadOnlyList<SymbolReferenceInfo.Types.SymbolReference> references,
+                                             int expectedDeclarationCount,
+                                             int assertedDeclarationLine,
+                                             params int[] assertedDeclarationLineReferences)
+        {
+            references.Where(x => x.Declaration != null).Should().HaveCount(expectedDeclarationCount);
```

Comment:
According to [this section](https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md#code-structure) of the coding style rules, `x.Declaration != null` should be changed into `x.Declaration is not null`.